src/System.Management.Automation/engine/parser/Compiler.cs

```diff
@@ -221,6 +221,8 @@ internal static class CachedReflectionInfo
 
         internal static readonly MethodInfo LanguagePrimitives_GetInvalidCastMessages =
             typeof(LanguagePrimitives).GetMethod(nameof(LanguagePrimitives.GetInvalidCastMessages), staticFlags);
+        internal static readonly MethodInfo LanguagePrimitives_IsNullLike =
+            typeof(LanguagePrimitives).GetMethod(nameof(LanguagePrimitives.IsNullLike), staticPublicFlags);
```

Comment:
staticPublicFlags or staticFlags?

Reply:
`IsNullLike` is public API, hence `staticPublicFlags`

https://github.com/PowerShell/PowerShell/blob/77ddec161417f98f8f2e57e9e9ef57d00cdc4c98/src/System.Management.Automation/engine/LanguagePrimitives.cs#L1074